morse_decoder.rb

```diff
@@ -0,0 +1,46 @@
+@dictionary = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_word(word)
+  @letters = word.split
+  @current_word = ''
+  @letters.each do |letter|
+    @current_word += decode_letter(letter)
+  end
```

Comment:
- [ ] Hi you are trying to call a function that doesn't exist `decode_letter` please kindly create the function and make your app work.